app/styles/main.scss

```diff
@@ -0,0 +1,207 @@
+@import "colors-fonts";
+
+:root {
+  --primary: var(--AppsilonBlue);
+  --primary-dark: var(--AppsilonNavi-strong-3);
+  --white: var(--AppsilonSnow);
+  --disabled: var(--AppsilonGray-strong-1);
+  --muted-dark: var(--AppsilonLightGray-weak-1);
+  --muted: var(--AppsilonGray-strong-1);
+}
+
+html,
+body {
+  height: 100vh;
+  color: var(--primary-dark);
+}
+
+.logos-wrapper {
+  display: flex;
+  align-items: end;
+  gap: 15px;
+
+  img.logo { height: 45px; }
+
+  &.dark {
+    display: none;
+  }
+
+  &.light {
+    display: flex;
+  }
+}
+
+// #####################################
+//
+//   _                    _
+//  | |                  | |
+//  | |__   ___  __ _  __| | ___ _ __
+//  | '_ \ / _ \/ _` |/ _` |/ _ \ '__|
+//  | | | |  __/ (_| | (_| |  __/ |
+//  |_| |_|\___|\__,_|\__,_|\___|_|
+//
+//
+//
+//  header
```

Comment:
I like the bigger text to find the section, but the small comment "header" seems unnecessary to me. I know it can help in finding "header" but our classes and divs have the same names (except single rules) so they can be found anyway. And, we don't use this for VARIABLES too.
But if you see value in this, would you consider center-aligning them so it's nice to see?

Reply:
It's a personal thing, I'll remove the small text!

Done in f86422f